### ISSUE TITLE
Snap package support

### ISSUE THREAD

**Issue by peteruithoven** [webtorrent/webtorrent-desktop#1071]
I've seen some related issues (https://github.com/feross/webtorrent-desktop/issues/629, https://github.com/feross/webtorrent-desktop/issues/190) but nothing to track publishing snap packages?

I [hear](https://github.com/electron-userland/electron-packager/issues/525#issuecomment-260109188) Electron hopes to support it this year: https://github.com/electron-userland/electron-builder/issues/509

More info:
http://snapcraft.io/
https://www.youtube.com/watch?v=j40tNL3t4gw

**Comment by feross** [comment]
Yes, snap does seem to be what the Linux community is standardizing on. I like like this a lot better than doing a bunch of extra work specifically for Ubuntu. If we produce a snap file, then we can submit to the Ubuntu Software Center easily.

**Comment by predr** [comment]
Well, *.deb didn't work for me #1529 had to use archive package. Updating is a pain. I was hoping there would be a snap package by now. If you are not aware they recently added:
1. Building directly from Github https://build.snapcraft.io/ and building on WIndows machines (Macs soon)
2. Distributing with snap://webtorrent-desktop for install buttons, opens in Ubuntu Software Center

**Comment by peteruithoven** [comment]
It should be getter easier with [Electron Forge](https://electronforge.io/) or [Electron builder](https://www.electron.build/)

**Comment by samuelpismel** [comment]
+1

**Comment by heidiwenger** [comment]
Was also looking for a snap package and found this. Is there any progress on a snap package yet?

Alan Pope (former Canonical) made this video on building a snap: https://www.youtube.com/watch?v=ikoUByA5BLU

Also there's one from his blog: https://popey.com/blog/2021/02/lets-go-snapping/

Thanks.
